core-services/ipi-deprovision/aws.sh

```diff
@@ -5,88 +5,197 @@ set -o pipefail
 
 trap finish TERM QUIT
 
+AWS_PROFILE="${AWS_PROFILE:-default}"
+CLUSTER_TTL="${CLUSTER_TTL:-12 hours}" # TTL for IAM users and clusters
```

Comment:
If a job runs for X hours, will those IAM users remove after 12 hours or X+12 hours ?